tests/pyximport/pyximport_pyimport_pythonpath.srctree

```diff
@@ -0,0 +1,26 @@
+PYTHON -c "import pyximport_test; pyximport_test.test()"
+
+########## mydir/mytest.py ##########
+
+import cython
+
+test_string = "MY_PY_TEST"
+compiled = cython.compiled
+
+######## pyximport_test.py ########
+
+import os
+import sys
+import pyximport
+
+sys.path.append('mydir')
+
+pyximport.DEBUG_IMPORT = True
```

Comment:
Is this needed?

Reply:
This allows printing to console on `_debug()` call. It is also in other tests, so I used it here too. I suppose it is for debugging purposes in case of failed test.